{{cookiecutter.project_name}}/pyproject.toml

```
@@ -74,3 +74,244 @@ testpaths = [
     "tests",
     "src/{{cookiecutter.project_name}}",
 ]
+{%- if cookiecutter.license_checker == "Y" -%}
```

This ensures that the default value (which is an empty string, but should be interpreted as "Y") also works.

```suggestion
{%- if cookiecutter.license_checker != "n" -%}
```